README.md

```diff
@@ -521,6 +521,7 @@ AWS Repos:
 Community Repos:
 
 * [Contribute](https://github.com/donnemartin/awesome-aws/blob/master/CONTRIBUTING.md)
+* [warrant](https://github.com/capless/warrant) - Python library for using AWS Cognito. 
```

Comment:
Can you make the displayed text capless/warrant for consistency?

Reply:
Yes, of course. I will make new PR, as I deleted a fork and can't make any change. Thank you for review.

Reply:
new PR https://github.com/donnemartin/awesome-aws/pull/112